src/feedback_forensics/app/callbacks.py

```diff
@@ -419,29 +342,23 @@ def _get_default_annotator_cols_config(data) -> str:
             target_models=[],
         )
 
-        # Get model annotator visible names using the utility function
-        _, avail_model_annotator_names = get_annotators_by_type(
-            votes_dict, MODEL_IDENTITY_ANNOTATOR_TYPE
-        )
-
-        avail_annotator_names = (
-            avail_datacol_annotator_names
-            + avail_principle_annotator_names
-            + avail_model_annotator_names
-        )
+        annotator_types = get_annotators_by_type(votes_dict)
+        all_annotator_names = []
+        for variant, annotators in annotator_types.items():
+            all_annotator_names.extend(annotators["visible_names"])
         return {
             inp["annotator_cols_dropdown"]: gr.Dropdown(
-                choices=avail_annotator_names,
+                choices=sorted(all_annotator_names),
```

Comment:
The avail datacol annotators may be spread before and after the others, which can be confusing and hard to find. We should also give them their own prefix like "Data: ..." so that they are their own list, are we doing that already?

Reply:
I have added prefixes to all annotator visible names in the latest commit.